### PR TITLE
Added color CIE to color HEX converter

### PR DESCRIPTION
**Pull request by marc2016**
Hello,

thank you for the awesome adapter :-)
I added a color CIE to color HEX converter to the adapter. All my Zigbee RBG lights have a color state in CIE format and most other adapter or smart home services using HEX codes for colors (e.g. ioBroker Jarvis or Google Home). To use the converter, the state must have the role "color.CIE" and the type "array".
I tested the adapter and fixed a few bugs. My extension has been running stable for two days :-)

### CODE REVIEW

**Comment by marc2016** [comment]
Hi.

I merged the new version in the pull request. Maybe you can look at it now :-)

**Comment by Scrounger** [comment]
Thank you.
Can you please add support for Admin 5 custom dialog - jsonCustom.json?

File: admin/jsonCustom.json [hunk 1]
```diff
@@ -456,6 +456,29 @@
                             }
                         }
                     }
+                },
+                "panel_expertSettings_colorCie": {
```
Comment (marc2016) [inline admin/jsonCustom.json:460]: This should BE the support for Admin 5 custom dialog? Or is something missing?

**Comment by Scrounger** [comment]
@marc2016:
perfect have not seen that's still implemented. Thank you!

**Pull request merged**